core/mutator/mutator.go

```diff
@@ -56,6 +56,13 @@ type Mutator interface {
 	Mutate(value, mutation proto.Message) (proto.Message, error)
 }
 
+// Mutation represents a change to a user, and associated data.
+type Mutation struct {
+	ID        int64
+	Mutation  *pb.Entry
```

Comment:
field `Mutation` within type `Mutation` feels a bit icky.  Could the field be `Entry` ?

Reply:
ah, now it is `Mutation` within `QueueMessage`. Less icky :)

Reply:
:-)